src/plugins/GUITestBase/src/GTUtilsWorkflowDesigner.cpp

```diff
@@ -951,7 +951,7 @@ void GTUtilsWorkflowDesigner::setDatasetInputFile(GUITestOpStatus &os, const QSt
 
     GTFileDialogUtils::TextInput t = pastePath ? GTFileDialogUtils::CopyPaste : GTFileDialogUtils::Typing;
 
-    GTFileDialogUtils *ob = new GTFileDialogUtils(os, filePath, GTGlobals::UseMouse, GTFileDialogUtils::Open, t);
+    GTFileDialogUtils *ob = new GTFileDialogUtils(os, filePath, GTGlobals::UseMouse, GTFileDialogUtils::Open, QString(), t);
```

Comment:
Can you move the filter parameter to the last in the parameters list please, so we won't be required to write a default value for it (looks like we have no single use of it today) ?

Reply:
OK